lib/pbench/test/unit/server/test_requests.py

```diff
@@ -57,19 +52,29 @@ def test_empty_url_path(client, caplog):
         assert caplog.records[0].levelname == "WARNING"
 
     @staticmethod
-    def test_bad_request(client, caplog, mocked_responses):
-        mocked_responses.add(
-            responses.POST,
-            "http://elasticsearch.example.com:7080/some_invalid_url",
-            status=400,
+    def test_bad_request(client, caplog, requests_mock):
+        requests_mock.post(
+            "http://elasticsearch.example.com:7080/some_invalid_url", status_code=400
         )
+        # mocked_responses.add(
+        #     responses.POST,
+        #     "http://elasticsearch.example.com:7080/some_invalid_url",
+        #     status=400,
+        # )
```

Comment:
commented code?

Reply:
Ah; that was Pete's old mock REST code. I commented it instead of deleting it, and then forgot about it.